W/Wayland/bundled/patches/Makefile_in.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile.in
```

Comment:
Patch proposed to upstream: https://gitlab.freedesktop.org/wayland/wayland/merge_requests/39

Reply:
You're awesome, Mosè.